### ISSUE TITLE
Minor issue: missing description for DialogContent

### ISSUE THREAD

**Issue by GSMPRANEETH** [coronasafe/care_fe#13700]
**Describe the bug**
While viewing valueset preview, in console it shows a error:
hook.js:608 Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.

**To Reproduce**

1. Go to 'Valueset'
2. Click on 'Valueset preview'
3. Scroll down to 'console'

**Expected behavior**
Console should not show this error

**Screenshots**

https://github.com/user-attachments/assets/2e4ba236-a7fe-43c1-9697-530fb19db54f

**Desktop (please complete the following information):**

- OS: Win 10
- Browser Opera
- Version Latest

**Additional context**
For any other pages that show this error, it should be fixed

---

### Instructions for Requesting Assignment:

To request assignment, please clearly outline your solution and timeline by commenting on the issue using the format below:

**Describe your solution clearly:**
Provide a detailed explanation of your proposed solution, including your approach, key implementation steps, and relevant examples or references. Mention any dependencies, assumptions, or risks you foresee that might affect your timeline or implementation.

**Expected Timeline:**
- End date: [Expected submission date of a completed Pull Request]

**Additional Context:**
Include any other relevant context, links, screenshots, or resources that support your proposed solution.

> 🚨 Your assignment may be unassigned if there is no activity or progress within the stated timeline unless communicated clearly and agreed upon.

**Comment by GSMPRANEETH** [comment]
@Jacobjeevan @rithviknishad @yash-learner 
I will add description for such components and validate for other pages too
ETA: 2 days

**Comment by Ishu813** [comment]
Hi @Jacobjeevan @rithviknishad
I would like to work on this issue
I would add `aria-describedby={undefined}` for {DialogContent}
ETA : EOD

**Comment by geyamrutha16** [comment]
@GSMPRANEETH , same happens  with `AlertDialogContent` while creating an encounter.

<img width="457" height="387" alt="Image" src="https://github.com/user-attachments/assets/f575795d-d03e-4077-9154-367b067600bc" />
